arangod/Aql/WindowExecutor.cpp

```diff
@@ -242,7 +242,7 @@ ExecutorState WindowExecutor::consumeInputRange(
       return state;
     }
   }
-  return inputRange.finalState();
+  return inputRange.upstreamState();
```

Comment:
This is just here because the bug-fix PR #15695 isn't merged yet, right?

Reply:
Yes correct